.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,7 +17,7 @@ fixes #X, mentioned in #Y
 *You can delete this section when you are submitting the pull request.* 
 
 * Please update this PR information accordingly. We'll use this as part of our release notes in monthly communications.
-* Please target your PR to "version-2" branch for v2 changes and "version-1" branch for v1 changes
+* Please target your PR to "version-3" branch for v3 changes, "version-2" branch for v2 changes and "version-1" branch for v1 changes
```

Comment:
Let's remove references to V2 and V1, as we are no longer supporting these branches or doing releases.

Reply:
No pb, will remove!

Reply:
Thank you! :)